conf/locale/babel_underscore.cfg

```diff
@@ -1,9 +1,5 @@
 # Use this configuration file for extracting strings from .underscore files.
-[underscore: cms/templates/**.underscore]
-input_encoding = utf-8
-[underscore: common/templates/**.underscore]
-input_encoding = utf-8
-[underscore: lms/templates/**.underscore]
+[underscore: **.underscore]
```

Comment:
I don't see any strings in the underscore.po file that are due to any `*fixture*` or `*mock*` file.  In fact, there's only one from those places, and it also appears in a file we do want extracted.
If it's OK with you, I'm going to punt on trying to exclude these directories.